src/Apps/Order/Utils/getStatusCopy.tsx

```diff
@@ -79,38 +79,41 @@ export const getStatusCopy = (order, logger?): StatusPageConfig => {
           isPrivateSaleOrder
         ),
         alertMessageTitle: isWireTransfer
-          ? "Please proceed with the wire transfer to complete your purchase"
+          ? "Please proceed with the wire transfer within 7 days to complete your purchase."
           : null,
         alertMessage: isWireTransfer ? (
           <>
-            <Text>
-              Please provide your proof of payment within 7 days. After this
-              period, your order will be eligible for cancellation by the
-              gallery.
-            </Text>
             <Spacer y={2} />
             <Text>
-              1. &nbsp; Find the order total and Artsy’s banking details below.
+              1. &nbsp; Find the total amount due and Artsy’s banking details
```

Comment:
Non-blocking. Curious if we have discussed using numbered bullet points so we don't need to add space to the copy.

Reply:
we haven't discussed this as far as I know – but good point!